src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,18 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
```

Comment:
Nice well commented code. Liked the sleek way you coded your comparator function. Nice job!